draft-ietf-lsr-isis-fast-flooding.xml

```diff
@@ -676,9 +676,42 @@
 
 		<section anchor="sec_determining_values" title="Determining values to be advertised in the Flooding Parameters TLV">
 			<t>The values that a receiver advertises do not need to be perfect. If the values are too low then the transmitter will not use the full bandwidth or available CPU resources. If the values are too high then the receiver may drop some LSPs during the first RTT and this loss will reduce the usable receive window and the protocol mechanisms will allow the adjacency to recover. Flooding slower than both nodes can support will hurt performance, as will consistently overloading the receiver.</t>
+
+			<section anchor="sec_determining_values_static" title="Static values">
 			<t>The values advertised need not be dynamic as feedback is provided by the acknowledgment of LSPs in SNP messages. Acknowledgments provide a feedback loop on how fast the LSPs are processed by the receiver. They also signal that the LSPs can be removed from receive window, explicitly signaling to the sender that more LSPs may be sent. By advertising relatively static parameters, we expect to produce overall flooding behavior similar to what might be achieved by manually configuring per-interface LSP rate-limiting on all interfaces in the network. The advertised values could be based, for example, on offline tests of the overall LSP-processing speed for a particular set of hardware and the number of interfaces configured for IS-IS. With such a formula, the values advertised in the Flooding Parameters TLV would only change when additional IS-IS interfaces are configured.</t>
+
+			<t>Static values are dependent on the CPU generation, class of router and network scaling, typically the number of adjacent neighbors.
+			Examples at the time of publication are provided below. LSP Burst Size could be in the range 5 to 20. From a router perspective, this value
+			typically depends on the buffer size on the I/O path from the packet forwarding engine to the control plane which is very platform dependent.

```

Comment:
Not sure what you mean by this distinction. Taking RFC7141 as a reasonable reference, may be changing to "length of the queue" ? (even though, personnaly, in this context I was not making a distinction between "buffer" and "queue" and RFC 7141 is using both (may be "queue" is implying an active management of the queue/buffer)

Reply:
"buffer size" to me means how large are the allocated buffers e.g., 1K, 2K, etc.
"number of buffers" seems more apt to me because I think you are taking about how many PDUs can be queued - given
that we have one PDU/buffer. (No fragmentation support in IS-IS)

Reply:
Thanks for the clarification. Text is talking about buffer in the dataplane between the incoming interface and the Route Processor. In the general case, it's not specific to IS-IS (although one could have a dedicated queue for IS-IS). The question seems about whether we measure queues in bytes or packets, which may be plateform dependent  https://www.[ietf.org/rfc/bcp/bcp41.html#section-4.1.1](https://www.ietf.org/rfc/bcp/bcp41.html#section-4.1.1). What about "amount of buffer" ?

Reply:
Maybe "queue(s) size(s)" (there might be different queues) might be more telling to implementers ? Socket buffer is clear enough but I agree buffer on its own, in a router, is ambiguous with a packet buffer

Reply:
Thanks done.